bridge/contracts/Federation.sol

```diff
@@ -96,22 +134,49 @@ contract Federation is Initializable, UpgradableOwnable {
         emit BridgeChanged(_bridge);
     }
 
+    function setBridgeNFT(address _bridgeNFT) external onlyOwner {
+      _setBridgeNFT(_bridgeNFT);
+    }
+
+    function _setBridgeNFT(address _bridgeNFT) internal {
+      require(_bridgeNFT != NULL_ADDRESS, "Federation: Empty NFT bridge");
+      bridgeNFT = INFTBridge(_bridgeNFT);
+      emit BridgeNFTChanged(_bridgeNFT);
+    }
+
+    function validateTransaction(bytes32 transactionId) internal view returns(bool) {
+      uint transactionCount = getTransactionCount(transactionId);
+      return transactionCount >= required && transactionCount >= members.length / 2 + 1;
+    }
+
+    /**
+      @notice Vote in a transaction, if it has enough votes it accepts the transfer
+      @dev Always return true
+      @param originalTokenAddress The address of the original token in the main chain
```

Comment:
I think this actually is the token address in the origin (which can be main or side, since the federation votes transactions coming from both sides from what I understand)

Reply:
It's actually always the address of the Token on the mainchain, see https://github.com/rsksmart/tokenbridge/blob/master/bridge/contracts/Bridge.sol#L417